### ISSUE TITLE
Bug on RP group-specific convolution

### ISSUE THREAD

**Issue by MarioRefoyo** [kdd2019-tapnet/tapnet#4]
The batch norm layer after the first group specific convolution should be also group specific, and not shared across groups. Doing so causes instabilities during training as the parameters of the batch norm can not adecuate across groups. 

Doing this when defining the group models solves the problem and yields results similar to those reported in the paper: 
`self.conv_1_models.append(
     nn.Sequential(*[
         nn.Conv1d(self.rp_dim, filters[0], kernel_size=kernels[0], dilation=dilation, stride=1,
                           padding=paddings[0]),
         nn.BatchNorm1d(filters[0])
     ])
)`
